docs/source/tertiary/regression-tests.rst

```diff
@@ -27,18 +29,17 @@ genome-wide association studies (GWAS).
 Linear regression
 =================
 
-``linear_regression_gwas`` performs a linear regression association test optimized for performance
+``linear_regression`` performs a linear regression association test optimized for performance
```

Comment:
Can we add some statements here as to what goes into the function and what comes out at a very high level? Also emphasis on the fact that multiple phenotypes can be processed in parallel is important. I know some concepts are explained in the API but I feel this paragraph needs some more high-level explanation to connect with the reader.